internal/changestream/eventmultiplexer/doc.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2025 Canonical Ltd. Licensed under the AGPLv3, see LICENCE file for
+// details.
+//
+// Package eventmultiplexer defines a worker that receives stream terms from the
+// internal/changestream/stream worker and sends them as events to the
+// appropriate event subscription.
+//
+// The eventmultiplexer sends these terms in an async manner, allowing all
+// subscribers to work on the same term at the same time.
+//
+// Terms are processed sequentially. This creates the potential for the
+// eventmultiplexer to get blocked by a subscriber, if a subscriber takes too
```

Comment:
No comma.

Reply:
How we breather here may differ, but fixed as you suggested.